src/main/resources/.gitignore

```diff
@@ -0,0 +1 @@
+api.properties
```

Comment:
Do we need to commit this to the repo?

Reply:
I thought it would be good to commit the folder just in case we need to add more resource files. The gitignore is there so the api.properties file (which stores the key) doesn't get uploaded.